README.md

```diff
@@ -54,7 +54,7 @@ As of version 2.0.1 (Nov. 2015) Picard requires Java 1.8 (jdk8u66). The last ver
 
 * To run a specific test, the command is:
 ```
-    ./gradlew test -Dtest.single=TestClassName 
+    ./gradlew legacyTest --tests "*TestClassName*"
```

Comment:
does this use the legacy parser? what about using the new parser?

Reply:
I added a barclay example too now.